src/components/ha-file-upload.ts

```diff
@@ -73,23 +76,27 @@ export class HaFileUpload extends LitElement {
   }
 
   public render(): TemplateResult {
+    const localize = this.localize || this.hass!.localize;
     return html`
       ${this.uploading
         ? html`<div class="container">
             <div class="uploading">
               <span class="header"
                 >${this.value
-                  ? this.hass?.localize(
-                      "ui.components.file-upload.uploading_name",
-                      { name: this._name }
+                  ? localize(
+                      "ui.panel.page-onboarding.restore.uploading_name",
```

Comment:
We should not use `page-onboarding` translations in generic elements, this translation will not be available outside onboarding